src/lib/hooks/useConnect/setupConnection.ts

```diff
@@ -34,13 +34,14 @@ export function setUpParams({
   sdkInitParams?: SendbirdChatInitParams;
 }): SendbirdChat {
   const params = {
-    appId,
     modules: [
       new GroupChannelModule(),
       new OpenChannelModule(),
     ],
     newInstance: true,
-    ...sdkInitParams,
+    ...(sdkInitParams ?? {}),
```

Comment:
@sravan-s So weird that 
Also, https://github.com/sendbird/sendbird-uikit-react/commit/a2f0541863b6f4ef5460af4a3792d893890f16cf  this commit from #692 is gone. So had to change this again 🤷‍♂️

Reply:
yeah weird